articles/overview/what-is-quantum-computing.md

```diff
@@ -26,7 +26,7 @@ The act of **measurement** changes a qubit. It goes from being in superposition
 
 Qubits can also be **entangled**: when we make a measurement of one entangled qubit, our knowledge of the state of the other is updated, even if the qubits are physically far apart.
 
```

Comment:
Word following colon is capitalized when it's a complete sentence:
**entangled**: When we make a...